### DISCUSSION TITLE
Map-agnostic, constant volume hitsound option

### DISCUSSION THREAD

**Discussion by Ashe** [ppy/osu#18654] [Ideas]
## Background

This is the one thing I always feel like I need whenever I play a map in osu which modifies the sounds of my hit volume. Since its been so long that I've been wanting this, I looked up why this hasn't been an option yet and found that it's [a really highly requested feature on the forums](https://osu.ppy.sh/community/forums/topics/193008?n=1).

For a game that can be customised so much, its so weird that this is the one thing that can't be changed. I fully respect the right for mappers be able to tweak the sounds for artistic purposes, but unless 'hitsound effects' somehow affects PP, we should be in control of how the game looks and sounds, especially when you consider things like accessibility. While one could learn to simply coexist with the idea that maps can change volume mid-map, at face value its weird having different volumes of hitsound depending on the map. 

This feature is probably fine for people with louder hitsounds, but for people like myself with subtle hitsounds, I want to hear it at full volume since the sound itself isn't too distracting. However, this feature means that mappers can be opinionated and change the volume level to suit their tastes, and as a player I have no idea what kinds of hitsounds I should be using for any given map for it to sound 'better' with volume changes.

## Proposal

Under skin/audio settings, make an option so that maps don't affect the volume level of your hitsounds. Regardless of map and sections within maps, your hitsound volume is constant and can be altered using the effects volume slider (or whatever volume slider changes hitsounds in the future).

**Comment by fadedlamp42** [comment]
100% agree, was searching around looking for a way to do this and sad that there's still no way. https://github.com/ppy/osu/pull/28733 seems like it has potential to fulfill this requirement, so I suggested it there.

**Comment by ItsPasi** [comment]
This still needs to happen.

**Comment by Fastnt-Rosmarin** [comment]
I want this sooo bad. It's crazy that this feature doesn't exist yet!
